chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -55,6 +55,7 @@ general.numMore                    = %S more…
 general.openPreferences				= Open Preferences
 general.keys.ctrlShift				= Ctrl+Shift+
 general.keys.cmdShift				= Cmd+Shift+
+general.dontShowAgain.button      = Don't Show Again
```

Comment:
Yeah, the point of putting this in `general.` is so that it's usable for, say, a checkbox label. So you'll just need to tweak the code that requests this.